1984/Assets/Scripts/Runtime/UI/Platformer/Movement.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    [SerializeField]
+    private float maxSpeed = 5.0f;
+    [SerializeField]
+    private float jumpForce = 15.0f;
+    private float idleGravity = 7.0f;
+    private float longJumpGravity = 4.0f;
+    private Rigidbody2D rigid2D;
+
+
+    [SerializeField]
+    private LayerMask groundLayer; 
+    private CapsuleCollider2D capsuleCollider2D;
+    private Vector3 footPosition; 
```

Comment:
```suggestion
    [Header("GroundCheck")]
    [SerializeField]
    private float maxSpeed = 5.0f;
    [SerializeField]
    private float jumpForce = 15.0f;
    private float idleGravity = 7.0f;
    private float longJumpGravity = 4.0f;
    private Rigidbody2D rigid2D;


    [SerializeField]
    private LayerMask groundLayer; 
    private CapsuleCollider2D capsuleCollider2D;
    private Vector3 footPosition; 
```

헤더 성격으로 묶어서 관리하는게 좋을 것 같아요

Reply:
따로 데이터 스크립터블 오브젝트로 빼서 관리하는 것도 좋은 방법입니다,,!

Reply:
데이터 스크립터블 오브젝트,,,,,,, 한번 고려해볼게요!!